dtmi/fii/fii/badge_apt-1.json

```diff
@@ -0,0 +1,38 @@
+{
+    "@id": "dtmi:fii:fii:badge_apt;1",

```

Comment:
@PoosaTsai , why the double `fii` for the `@id` components?  Seems the root `@id` could be `dtmi:fii:badge_apt;1`?

Reply:
Hi @digimaun
I am Berry Tsai who from Microsoft Taiwan for Fii's technical support.
First fii is company name. About the second fii that because FII are ODM and own brand role that may have some devices manage by OEM customers. Fii R&D would like to use third column to identify who manage devices. For this ID "dtmi:fii:fii:badge_apt;1" mean device managed by fii.

Reply:
Thanks for the clarification.